sdk-libs/sdk/README.md

```
@@ -0,0 +1,151 @@
+
+For full program examples, see the [Program Examples](https://github.com/Lightprotocol/program-examples).
+For detailed documentation, visit [zkcompression.com](https://www.zkcompression.com/).
+For pinocchio solana program development see [`light-sdk-pinocchio`](https://docs.rs/light-sdk-pinocchio).
+For rust client development see [`light-client`](https://docs.rs/light-client).
+For rust program testing see [`light-program-test`](https://docs.rs/light-program-test).
+For local test validator with light system programs see [Light CLI](https://www.npmjs.com/package/@lightprotocol/zk-compression-cli).
+
+### Difference to Light-Accounts (Light-PDA)
+Light-PDA's are Solana accounts with sponsored rent-exemption.
+There is no proof required for interactions with Light-PDA's which makes
+them suitable for Defi Usecases. Compressed PDA's don't require rent-exemption, 
```

markdownlint-cli2 (0.20.0)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)